File: Python2/to_lower_case.py
class Solution(object):
    def toLowerCase(self, str):
        """
        :type str: str
        :rtype: str
        """
        import StringIO
        output = StringIO.StringIO()
        Z = ord('Z')
        A = ord('A')
        for ch in str:
            ascii_code = ord(ch)
            if ascii_code <= Z and ascii_code >= A:
                ch = chr(ascii_code+32)
            output.write(ch)
        return output.getvalue()


if __name__ == '__main__':
    s = 'Hello'
    sol = Solution()
    print sol.toLowerCase(s)

File: C++/to_lower_case.cpp
#include "./to_lower_case.h"
#include <iostream>

using namespace std;

string Solution::toLowerCase(string str) {
    for (int i = 0; i<str.length(); ++i) {
        if (str[i] <= 'Z' && str[i] >= 'A') {
            str[i] += 32;
        }
    }
    return str;
}


int main() {
    string input = "Hello";
    Solution* sol = new Solution();
    string output = sol->toLowerCase(input);
    delete sol;
    cout << output << "\n";
    return 0;
}